### ISSUE TITLE
Scrollbar is shown on every toggle

### ISSUE THREAD

**Issue by sindresorhus** [binaryage/totalterminal#79]
In the latest version of TotalTerminal on Mavericks when I toggle the terminal using a hotkey the scrollbar is faded in, then faded out. This is a tiny bit annoying. Is there any way that could not happen?

**Comment by sindresorhus** [comment]
@darwin ping :)

**Comment by darwin** [comment]
There is one other annoying thing. Transferring focus may be a bit delayed during sliding and one can miss a keystroke. 

I'm going to look into both issues.

**Comment by darwin** [comment]
I did some investigation and problem is somewhere deeper. This scrollbar fading is triggered when terminal window is brought front when it was previously hidden (Cocoa's `[window orderFront:nil]` call). I think in this case Terminal's code tries to resize window or repositions/recreates ScrollView inside main window which triggers this effect. I observed other apps and for example latest beta of XCode does this when you select different source files by mouse clicking. Source Editor shows scrollbars in similar fashion if source file is big enough to have vertical scrollbar.

Temporary workaround would is to set (actually this is my system-wide setting, that is why I didn't know about this issue):

```
defaults write com.apple.Terminal AppleShowScrollBars Always
```

Looking for a better solution.
